src/services/manual-status-client.ts

```diff
@@ -16,8 +16,8 @@ export class HttpManualStatusClient implements ManualStatusClient {
     try {
       const response = await this.axios.request({
         method: 'GET',
-        url: 'http://toggle.fromdoppler.com/webapp/webapp-status.json?' + new Date().getTime(),
-        headers: { 'Cache-Control': 'no-store, max-age=0' },
+        url: 'https://toggle.fromdoppler.com/webapp/webapp-status.json?' + new Date().getTime(),
```

Comment:
Https endpoint is not working yet, will you talk with @dnoya?

Reply:
Si ya hablé, luego me dijo que lo miraba, entonces dejo esto en suspenso y voy viendo otras cosas mientras.